miqa/core/rest/project.py

```diff
@@ -172,7 +172,7 @@ def settings_(self, request, **kwargs):
         request_body=no_body,
         responses={204: 'Import succeeded.'},
     )
-    @project_permission_required(superuser_access=True)
+    @project_permission_required()
```

Comment:
Not required, but you can pass in `review_access=True` if you think collaborators should not be given the same power of Import

Reply:
It sounded like Killian wants everyone to be able to import, so I'm waiting on confirmation of that.

Reply:
radio silence so far. I'd say we merge this as-is and we can lock it down later if appropriate.

Reply:
In the interest of only opening it up as much as we need to, can we get away with just opening it to review_access? If so, that'd be my vote.